### PR TITLE
Add the landscape picture on the community page

### PR DESCRIPTION
**Pull request by m-linner-ericsson**
### Applicable Issues
In order to have a better overview of the different components in the Eiffel Community, this PR aims to provide a landscape picture. To see it in action please see the [preview](https://m-linner-ericsson.github.io/eiffel-community.github.io/).

**Note** This PR originated as https://github.com/eiffel-community/eiffel-sepia/pull/13

### Description of the Change
Add a landscape picture of the different components in the Eiffel community to the Community page

To edit the picture open it via draw.io (where you can interact directly with GitHub)

### Alternate Designs
We could use the landscape functionality use by Linux foundation https://landscape.cd.foundation/ or https://landscape.cncf.io/ but the effort of getting one of those up and running was considered too much.

### Benefits
One can see an overview of what components we have in the Eiffel Community

### Possible Drawbacks
None that I can think of

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Mattias Linner <mattias.linner@ericsson.com>

### CODE REVIEW

File: community.html [hunk 1]
```diff
@@ -27,7 +27,12 @@ <h1 class="section-heading">Contact</h1>
     <section>
       <div>
         <h1 class="section-heading">Browse the eco-system</h1>
-        <p class="centered-text">Browse the Eiffel repositories in the <a href="https://github.com/eiffel-community">eiffel-community organization on GitHub</a>.</p>
+        The Eiffel eco-system contains projects that use the Eiffel protocol, both sending and consuming. For details on these projects check them out in the <a href="https://github.com/eiffel-community">eiffel-community organization on GitHub</a>.
```
Comment (magnusbaeck) [inline community.html:30]: Could we change eco-system to ecosystem on both lines?

**Review by magnusbaeck — CHANGES_REQUESTED** [review]
The landscape image looks great!

Just a tiny language nit (inline), but also a question: Is there any particular reason for keeping the PNG? The more files to update when the original source changes the more files can get out of sync. The landscape will change fairly often so I think updating it should be very easy.

I hope we at some point can update the SVG to contain hyperlinks to the repos listed there.

**Review by t-persson — APPROVED** [review]

File: community.html (hunk 1 above)
Comment (m-linner-ericsson) [reply community.html]: Fixed in b614d51

**Comment by m-linner-ericsson** [comment]
> The landscape image looks great!
> 
> Just a tiny language nit (inline), but also a question: Is there any particular reason for keeping the PNG? The more files to update when the original source changes the more files can get out of sync. The landscape will change fairly often so I think updating it should be very easy.
> 
> I hope we at some point can update the SVG to contain hyperlinks to the repos listed there.

Valid question! If you look carefully at the two pictures, you can see that they do not look the same. For example, they have different fonts. I find the PNG version to be better looking and could probably remove the SVG version if I am not using it.

I have not experimented with links in the picture but it could be that I then have to convert the picture to Inkscape - which would be one way to go.

**Review by magnusbaeck — APPROVED** [review]
We've discussed image formats back and forth enough now so let's get this in. We can change our minds later.

> Valid question! If you look carefully at the two pictures, you can see that they do not look the same. For example, they have different fonts. I find the PNG version to be better looking and could probably remove the SVG version if I am not using it.

They're _slightly_ different, but on my machine it's actually the SVG that looks better :-) The inter-letter spacing in e.g. "Specification & Architecture" looks pretty bad in the PNG but looks fine in the SVG.

> I have not experimented with links in the picture but it could be that I then have to convert the picture to Inkscape - which would be one way to go.

Okay, so draw.io can't add hyperlinks?

**Review by e-backmark-ericsson — APPROVED** [review]

**Pull request merged**